grammars/tree-sitter-ruby.cson

```diff
@@ -120,6 +120,7 @@ scopes:
     {match: '^[A-Z_]+$', scopes: 'constant'}
```

Comment:
This should be changed to `variable.constant` if we want to match the textmate grammar and different highlighting for constant identifiers and constant literals.

Reply:
I think let's go ahead and change it.